pkg/operator/baremetal_pod.go

```diff
@@ -15,10 +18,22 @@ import (
 )
 
 const (
-	baremetalConfigmap    = "metal3-config"
-	baremetalSharedVolume = "metal3-shared"
-	baremetalSecretName   = "metal3-mariadb-password"
-	baremetalSecretKey    = "password"
+	baremetalConfigmap         = "metal3-config"
+	baremetalSharedVolume      = "metal3-shared"
+	baremetalSecretName        = "metal3-mariadb-password"
+	baremetalSecretKey         = "password"
+	ironicCredentialsVolume    = "metal3-ironic-basic-auth"
+	inspectorCredentialsVolume = "metal3-inspector-basic-auth"
+	ironicUsernameKey          = "username"
+	ironicPasswordKey          = "password"
+	ironicHtpasswdKey          = "htpasswd"
+	ironicConfigKey            = "auth-config"
+	ironicSecretName           = "metal3-ironic-password"
+	ironicUsername             = "ironic-user"
+	inspectorSecretName        = "metal3-ironic-inspector-password"
+	inspectorUsername          = "inspector-user"
+	metal3AuthRootDir          = "/auth"
+	HtpasswdEnvVar             = "HTTP_BASIC_HTPASSWD"
```

Comment:
HtpasswdEnvVar  should be htPasswdEnvVar ?

Reply:
[`.htpasswd`](https://en.wikipedia.org/wiki/.htpasswd) is the name of the file that Apache reads credentials from (whose format we are borrowing) and [`htpasswd`](https://httpd.apache.org/docs/2.4/en/programs/htpasswd.html) is the name of the utility that Apache provides for writing them. It's literally anybody's guess as to how this ought to be capitalised if it were not a filename :laughing: so I decided not to try.

Reply:
Maybe `htpasswdEnvVar` then, but the H shouldn't be capitalized, do we have a reason to export it?

Reply:
Ugh, sorry, complete brain explosion there. Y'all are correct, obviously.